torchvision/csrc/custom_ops/custom_ops.cpp

```diff
@@ -0,0 +1,13 @@
+#include <torch/script.h>
+
+#include "nms.h"
+#include "ROIAlign.h"
+#include "ROIPool.h"
+
+using namespace at;
+
+static auto registry =
+  torch::jit::RegisterOperators()
+    .op("torchvision::nms", &nms)
+    .op("torchvision::roi_align_forward(Tensor input, Tensor rois, float spatial_scale, int pooled_height, int pooled_width, int sampling_ratio) -> Tensor", &ROIAlign_forward)
```

Comment:
@suo will there be a way of registering an operation with backprop via the `RegisterOperators`?